doc/implementation/core-physics/stepping.rst

```diff
@@ -58,8 +58,9 @@ single "action" class can, for example, allocate states at the beginning of the
 run and execute once per step.
 
 There are currently two different actions that act as extension points to the
-stepping loop: :cpp:class:`BeginRunActionInterface` is called once per event
-(or set of simultaneously initialized events), and :cpp:class:`StepActionInterface`
+stepping loop: :cpp:class:`celeritas::BeginRunActionInterface` is called once
+per event
+(or set of simultaneously initialized events), and :cpp:class:`celeritas::StepActionInterface`
```

Comment:
```suggestion
stepping loop: :cpp:class:`celeritas::BeginRunActionInterface` is called once
per event (or set of simultaneously initialized events), and :cpp:class:`celeritas::StepActionInterface`
```

Reply:
In general do we want to reformat whole paragraphs of docs when one line is changed? I tend to prefer uneven lines in the source code that give a better `git blame`.